src/domain/Reservation/components/ReservationCalendar.tsx

```diff
@@ -0,0 +1,252 @@
+// dayjs-calendar/Calendar.jsx
+import dayjs from 'dayjs';
+import isSameOrBefore from 'dayjs/plugin/isSameOrBefore'; // 반복문을 위한 플러그인
+import React, { useMemo, useState } from 'react';
```

Comment:
React가 불필요하게 import 되어 있습니다!
```suggestion
import { useMemo, useState } from 'react';
```